containers/java-postgres/test-project/src/main/java/com/mycompany/app/App.java

```diff
@@ -0,0 +1,34 @@
+/*-------------------------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+ *-------------------------------------------------------------------------------------------------------------*/
+
+package com.mycompany.app;
+
+import static javax.xml.XMLConstants.XML_NS_PREFIX;
+import java.io.*;
+import java.net.*;
+
+
+
+public class App 
+{
+    public static void main( String[] args )
+    {
```

Comment:
How come we're not following the pattern in `dotnet-mssql`? The `dotnet-mssql` app establishes that the database is operational and returns the names of the available databases. 

See: `https://github.com/microsoft/vscode-dev-containers/blob/d7c373f5b497ce8dba496e47b77a25ff34d0cf9e/containers/dotnet-mssql/test-project/Program.cs#L20`

Reply:
On further reflection, @dnastrain and I have agreed to try and quickly run an operational test within `App.Java` to see if the PostgreSQL database is up and running.

Reply:
Done!